src/components/registration/steps.jsx

```diff
@@ -879,7 +880,11 @@ module.exports = {
             };
         },
         onNextStep: function () {
-            this.props.onNextStep();
+            this.props.onNextStep({
+                user: {
+                    username: this.props.studentUsername
```

Comment:
Yeah – this is somewhat left over from when I was trying a previous approach of re-using 'notEqualsField`, and having the username be a field that was used in analysis (that was to try and avoid using `notEquals` in two places because of formsy constraints). I think that removing this and just using the prop in the next step is better too.